pywb/static/wombat.js

```diff
@@ -1326,6 +1330,144 @@ var _WBWombat = function($wbwindow, wbinfo) {
     }
 
     //============================================
+    function initPreserveWorker() {
+        if (!wbUsePresWorker) {
+            return;
+        }
+
+        var Preserver = (function(Worker) {
+            function PWorker(prefix, mod) {
+                if (!(this instanceof PWorker)) {
+                    return new PWorker(prefix, mod);
+                }
+                if ($wbwindow === $wbwindow.__WB_replay_top) {
+                    // we are top and can will own this worker
+                    // setup URL for the kewl case
+                    var workerURL = wbinfo.static_prefix +
+                        'wombatPreservationWorker.js?prefix=' +
+                        encodeURIComponent(prefix) + '&mod=' +
+                        encodeURIComponent(mod);
+                    this.worker = new Worker(workerURL);
+                    if (typeof $wbwindow.URL === 'undefined') {
```

Comment:
Maybe this should just be the default, to avoid the multiple init paths? (I wonder which browsers support workers and not window.URL..) Just trying to remove variation :)

Reply:
It would seem that safari is the only browser that might give us trouble is safari (`searchParams` available in v10.1)